qemu/tests/sr_iov_hotplug.py

```diff
@@ -319,7 +380,14 @@ def _device_removed():
                 msg += " repeat %d" % (j + 1)
                 error_context.context(msg, logging.info)
                 pci_del(-(pci_num + 1))
+
+            # cleanup network script after hot deleting pci device
+            clean_network_scripts()
```

Comment:
Not necessary, since L388 will always be executed.

Reply:
This is necessary because once hot deleting is done, network script should be cleaned for next iteration to work fine.